package.json

```diff
@@ -111,7 +111,7 @@
     ],
     "macos": {
       "identifier": "io.balena.cli",
-      "sign": "Developer ID Installer: Balena Ltd (66H43P8FRG)"
+      "sign": "\"Developer ID Installer: Balena Ltd (66H43P8FRG)\""
```

Comment:
With this we do not require the patch to escape the parameters for signing mac, we rather pass a escaped string